Description: Remove unavailable features
 As we use Debian packaged libzstd not the bundled, some features are
 not (can not be) enabled at build time and unavailable, or made no-ops:
 .
 - debug, fat-lto, no_asm, thin, thin-lto: removed in zstd-sys thus zstd-safe
 - pkg-config, legacy: no-op in zstd-sys thus zstd-safe
 - bindgen: non-existent in zstd-sys, removed in zstd-safe
 - wasm: no-op in original code
Last-Update: 2023-02-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: zstd/Cargo.toml
===================================================================
--- zstd.orig/Cargo.toml
+++ zstd/Cargo.toml
@@ -53,8 +53,6 @@ repository = "gyscos/zstd-rs"
 
 [features]
 arrays = ["zstd-safe/arrays"]
-bindgen = ["zstd-safe/bindgen"]
-debug = ["zstd-safe/debug"]
 default = [
     "legacy",
     "arrays",
@@ -62,15 +60,10 @@ default = [
 ]
 doc-cfg = []
 experimental = ["zstd-safe/experimental"]
-fat-lto = ["zstd-safe/fat-lto"]
 legacy = ["zstd-safe/legacy"]
-no_asm = ["zstd-safe/no_asm"]
 pkg-config = ["zstd-safe/pkg-config"]
-thin = ["zstd-safe/thin"]
-thin-lto = ["zstd-safe/thin-lto"]
 wasm = []
 zdict_builder = ["zstd-safe/zdict_builder"]
-zstdmt = ["zstd-safe/zstdmt"]
 
 [lib]
 name = "zstd"
